application/backup.sh

```diff
@@ -60,6 +60,20 @@ for dbName in ${DB_NAMES}; do
     else
         echo "mysql-backup-restore: Copy backup to ${S3_BUCKET} of ${dbName} completed in $(expr ${end} - ${start}) seconds."
     fi
+
```

Comment:
I thought you were going to encrypt things. Where is that code?

Reply:
I realized that @forevermatt and I had decided to get the backups to B2 in place and create an issue to add encryption.